src/main/scala/is/hail/io/plink/PlinkLoader.scala

```diff
@@ -202,6 +201,9 @@ object PlinkLoader {
     nPartitions: Option[Int] = None, a2Reference: Boolean = true, gr: GenomeReference = GenomeReference.defaultReference,
     contigRecoding: Map[String, String] = Map.empty[String, String], dropChr0: Boolean = false): MatrixTable = {
     val (sampleInfo, signature) = parseFam(famPath, ffConfig, hc.hadoopConf)
```

Comment:
parseFam should just return an Array[Annotation] with the right (full) schema including the sampleId.